rules/windows/execution_initial_access_via_msc_file.toml

```
@@ -0,0 +1,98 @@
+[metadata]
+creation_date = "2024/05/12"
+integration = ["endpoint", "windows", "system"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2024/05/12"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of a child process from a Microsoft Common Console file. Adversaries may embed a malicious command
+in an MSC file in order to trick victims into executing malicious commands.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.process-*", "logs-windows.*", "endgame-*", "logs-system.security*"]
```

```suggestion
index = ["winlogbeat-*", "logs-endpoint.events.process-*", "logs-windows.sysmon_operational-*", "endgame-*"]
```

Security logs don't have `process.parent.args`